godtossing/Pos/Binary/GodTossing/Core.hs

```diff
@@ -20,16 +20,14 @@ import           Pos.Ssc.GodTossing.Core.Types (Commitment (..), CommitmentsMap
 import           Serokell.Util                 (allDistinct)
 
 instance Bi Commitment where
-  encode Commitment{..} = encodeListLen 3 <> encode commShares
```

Comment:
There was no bug. In the previous version of PVSS we had three fields in a commitment, now we have two.